addons/account/tests/test_account_tax.py

```diff
@@ -3,12 +3,44 @@
 from odoo import Command
 from odoo.addons.account.tests.common import AccountTestInvoicingCommon
 from odoo.tests import tagged
-from odoo.exceptions import UserError, ValidationError
+from odoo.exceptions import UserError
 
 
 @tagged('post_install', '-at_install')
 class TestAccountTax(AccountTestInvoicingCommon):
 
+    @classmethod
+    def setUpClass(cls):
```

Comment:
Instead of extending the `setUpClass`, I think it would be easier to understand each test if this was done in a helper function that is called before checking that the tax `is_used`.